gst/nnstreamer/nnstreamer_plugin_api_filter.h

```diff
@@ -37,6 +37,7 @@ typedef struct _GstTensorFilterProperties
   const char *fwname; /**< The name of NN Framework */
   int fw_opened; /**< TRUE IF open() is called or tried. Use int instead of gboolean because this is refered by custom plugins. */
   const char *model_file; /**< Filepath to the model file (as an argument for NNFW). char instead of gchar for non-glib custom plugins */
+  const char *model_file_init; /**< Filepath to the init model file (as an argument for NNFW). Some frameworks need this file to initialize the graph(caffe, caffe2) */
```

Comment:
Because there is a possibility that other subplugins may want to use this, let's name it ```model_file_secondary``` or ```model_file_sub```.

Reply:
it's changed